tests/artist-read.test.js

```diff
@@ -0,0 +1,68 @@
+// tests/artist-read.test.js
+const { expect } = require('chai')
+const request = require('supertest')
+const getDb = require('../src/services/db')
+const app = require('../src/app')
+
+describe('read artist', () => {
+  let db
+  let artists
+  // establishing connection to db before each test
```

Comment:
Again, comment should be removed as it doesnt provide anything code doesnt already tell us